syntax = "proto3";

package grpc_services;

// Message de demande
message ItineraryRequest {
  string start_location = 1;
  string end_location = 2;
}

// Message de r√©ponse
message ItineraryResponse {
  repeated string steps = 1;
}

// Service gRPC
service ItineraryService {
  rpc GetItinerary(ItineraryRequest) returns (ItineraryResponse);
}
